include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -57,94 +57,164 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
-  void
-  finish_time_step_particles();
-
-  // Evaluate the L2 error on the computational domain if an analytical solution
-  // is given.
-  double
-  calculate_L2_error_particles();
-
-  virtual void
-  postprocess_fd(bool firstIter) override;
-
-  // Allow a refinement around each of the particles.
-  void
-  refine_ib();
-
-  // Return a bool to define if a cell is cut by a IB particle, the Id fo the particle that cut it if it's the case and the local dof of the cell for later use
-  std::tuple<bool,unsigned int, std::vector<types::global_dof_index>>
-  cell_cut(const typename DoFHandler<dim>::active_cell_iterator &cell,std::vector<types::global_dof_index> &local_dof_indices, std::map<types::global_dof_index, Point<dim>> &support_points);
-
-  // Return the cell around a point based on a initial guess of a closed cell (look in the neighbors of this cell)
-  typename DoFHandler<dim>::active_cell_iterator
-  find_cell_around_point_with_neighbors(const typename DoFHandler<dim>::active_cell_iterator &cell,
-                                     Point<dim>             point);
-
-  // Return a vector of cells around a cell including vertex neighbors
-  std::vector<typename DoFHandler<dim>::active_cell_iterator>
-  find_cells_around_cell(const typename DoFHandler<dim>::active_cell_iterator &cell);
-
-  std::vector<typename DoFHandler<dim>::active_cell_iterator>
-  find_face_neighbors_around_cell(const typename DoFHandler<dim>::active_cell_iterator &cell);
-
-  bool
-  check_if_cells_share_vertex(const typename DoFHandler<dim>::active_cell_iterator &cell_1,const typename DoFHandler<dim>::active_cell_iterator &cell_2);
-
-  // clear all the line of dof even if the dof is not owned but it is ghost
-  void
-  clear_line_in_matrix(const typename DoFHandler<dim>::active_cell_iterator &cell, unsigned int dof_index);
-
-
-  // Modified version of assemble_matrix_and_rhs to include the presence of
-  // extra steps.
-  void
-  assemble_matrix_and_rhs(
-    const Parameters::SimulationControl::TimeSteppingMethod
-      time_stepping_method) override;
-
-  void
-  assemble_rhs(const Parameters::SimulationControl::TimeSteppingMethod
-                 time_stepping_method) override;
-
   /**
-   * Members
+  * @brief Calculate the coefficients required for BDF integration of order n.
+  * The coefficients are determined through a recursion algorithm.
+  * The algorithm is taken from : Hay, Alexander, et al. "hp-Adaptive time
+   * integration based on the BDF for viscous flows." Journal of Computational
+   * Physics 291 (2015): 151-176.
+   *
+   * @param order The order of the BDF method. The BDF method of order n requires n+1 arrays
+   *
+   * @param time_steps a vector containing all the time steps. The time steps should be in reverse order.
+   * For example, if the method is a BDF2, it uses three values for the time : (t)
+   * (t-dt_1) and (t-dt_2). Thus the time step vector should contain dt_1 and
+   * dt_2.
```

Comment:
Why this is here? Maybe a copy-paste mistake?
```suggestion
```

Reply:
Indeed !